news/2023/2023-10-08-osu-mania-world-cup-2023.md

```diff
@@ -0,0 +1,165 @@
+
+> **myucchii:** The match against Vietnam was overall very stressful to me. Even though I was feeling good, and the team was performing very well, we weren't getting the points we were expecting to get in the match. I was also really nervous because this was my first time getting into the semifinals. We ended up getting to the tiebreaker, and I never checked the scoreboard at any point; I was only focused on comboing and trying my hardest to get my team to win. The moment I saw the final message from the referee, saying that we won the match, was probably one of the best moments I've ever experienced in my entire osu!mania career as a player, to the point where I cried because of how happy I was. Every time I scroll through my maps in the osu! menu and see HALL, I just think of that moment.
+
+*Your team managed to snatch an impressive top 8 run, making it to the second round of Semifinals. Do you think that placement is “accurate”, for lack of a better term?*
+
+> **myucchii:** Honestly, the teams’ placement this year was very fair. Going into this tournament, we thought the difficulty increase in the mappools could potentially give us more of a chance to put up a good fight against the top few teams, and I’d say that we succeeded, particularly on the LN and hybrid maps.
+
+*Tell us about your vision of the future for Team Chile. Do you have plans to continue to coach or develop players from your team to eventually reach the podium?*
+
+> **myucchii:** I’ve always had plans to continue assisting in fostering the competitive 4K environment within the Chilean community. I’m very glad to already have been able to assist players along their journeys and witness their potential manifest in real time; for instance, MyAngelKokomi and [LS]bambi fnf, who had never even played in 4K MWC before, proved themselves to be two of the most capable tournament players Team Chile has seen. With them and even more players on the horizon, top 3 feels ever so slightly closer. It’s still my ultimate goal to achieve in this game, and I won’t ever stop trying to encourage my fellow Chilean players to share in that dream.
+
+*Any closing thoughts?*
+
+> **myucchii:** 4K MWC 2023 was very memorable to me; it was by far the best tournament I’ve taken part in, and I cannot express my gratitude enough for the mappoolers, referees, casters, and everyone else on the staff team for making this such an incredible experience. You all deserve the world and more credit for your work through this tedious process, which will always have a place in my heart forever. Thank you so much.
+
+### Nepijin
+
+*Congratulations to Team United States on the first place in 4K MWC 2023! Nepijin, you’ve been playing in 4K MWC since 2021, as well as taking the leadership role as Team Captain for the past two years. You must have had quite the journey to get here; how are you feeling?*
```

Comment:
```suggestion
*Congratulations to Team United States on the first place in 4K MWC 2023! [Nepijin](https://osu.ppy.sh/users/11734610), you’ve been playing in 4K MWC since 2021, as well as taking the leadership role as Team Captain for the past two years. You must have had quite the journey to get here; how are you feeling?*
```